### PR TITLE
Core UI 2

### PR DESCRIPTION
**Pull request by tan-nate**

### CODE REVIEW

File: backend/db/reference_csv/urltype.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
+id,name,do_not_publish,is_email
+2,Check if I'm registered,N,N
+3,Online Voter Registration,N,N
+4,Find my polling place,N,N
```
Comment (jmensch1) [inline backend/db/reference_csv/urltype.csv:4]: hi @tan-nate, are all of these files in `/backend/db/reference_csv` just copies of the files in the `/backend/db/example_data.zip`? If so, maybe we could just `.gitignore` the `/backend/db/reference_csv` folder. Seems unnecessary to have two copies of all of this in version control.

File: client/web/src/redux/store.js [hunk 2]
```diff
@@ -2,9 +2,10 @@ import { createStore, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
 import { composeWithDevTools } from 'redux-devtools-extension';
 
+import preloadedState from './preloadedState';
```
Comment (jmensch1) [inline client/web/src/redux/store.js:5]: interesting...what's the state that we're loading here?

Comment (jmensch1) [reply client/web/src/redux/store.js:5]: oh, I just saw above it's the dummy data.

File: backend/db/reference_csv/urltype.csv (hunk 1 above)
Comment (tan-nate) [reply backend/db/reference_csv/urltype.csv:4]: ok cool, just added that folder to the gitignore

**Review by jmensch1 — APPROVED** [review]
looks good! I just added one commit removing those csv from version control. They had been committed previously so adding them to the .gitignore didn't work.

**Pull request merged**